spec/integration/gds_sso_spec.rb

```diff
@@ -5,6 +5,10 @@
     allow(Settings).to receive(:auth_provider).and_return("gds_sso")
   end
 
+  after do
+    OmniAuth.config.test_mode = false
+  end
```

Comment:
```suggestion
	let(:test_mode) { false }

  after do
    OmniAuth.config.test_mode = test_mode
  end
```

We can then set the test mode to true when we need it

Reply:
We don't want it to be `true` when the tests in the scenario are done, because that could interfere with tests in other scenarios. Do you mean changing it for the `before` block?